.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,102 @@
+# NOTE: This workflow is overkill for most R packages
+# check-standard.yaml is likely a better choice
+# usethis::use_github_action("check-standard") will install it.
```

Comment:
I don't know anything about this, but it sounds like we should at least consider the "check-standard" version of this.

Reply:
Thanks for promoting simplicity. Unfortunately the [check-standard.yml](https://github.com/r-lib/actions/blob/master/examples/check-standard.yaml) checks only with R-release and R-devel, but this repo supports R >= 3.5. Our other workflow ([run-integration-test.yaml](https://github.com/2DegreesInvesting/PACTA_analysis/blob/current_web_functionality/.github/workflows/run-integration-test.yaml)) also checks the full range from R 3.5 to R-devel.

The complexity doesn't worry me; this workflow is well maintained and I follow it closely because I use it in all the packages we have on CRAN.